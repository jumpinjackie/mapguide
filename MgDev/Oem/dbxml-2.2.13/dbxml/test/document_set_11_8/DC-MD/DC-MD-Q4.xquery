for $a in collection("11.8.1.dbxml")/order[@id="2"]
return
    $a/order_lines/order_line[1]
