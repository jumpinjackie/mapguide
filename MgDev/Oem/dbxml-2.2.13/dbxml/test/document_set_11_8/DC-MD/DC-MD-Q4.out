<order_line id="1"><item_id>611</item_id>
<quantity_of_item>44</quantity_of_item>
<discount_rate>0.02</discount_rate>
<special_instructions>m3,GZTY#N5Jmvo]o*7.uTMN=K4uaz cngdAQd}RiS5SM8$GTxXb{y7-!Rd|D,v~D5;}GG2U)TAyZQ1s</special_instructions>
</order_line>